components/Hero.tsx

```diff
@@ -1,22 +1,28 @@
+'use client'
```

Comment:
Like I mentioned in my other comment, we could make this a Server component, but then we couldn't use `usePathname` and would have to prop drill the `lang` all the way down to where you'd want to pass it to `useTranslations`, since `usePathname` doesn't work for Server components.

Though I agree with you that the app dir approach is a bit of a pain (at least in its current state), let's just roll with it for now.